examples/js/renderers/CSS3DRenderer.js

```diff
@@ -58,7 +59,6 @@ THREE.CSS3DRenderer = function () {
 	var cameraElement = document.createElement( 'div' );
 
 	cameraElement.style.WebkitTransformStyle = 'preserve-3d';
-	cameraElement.style.MozTransformStyle = 'preserve-3d';
```

Comment:
I guess we don't need the vendor prefix anymore since Fireofx securely supports the default `transformStyle`?